libs/wire-api/src/Wire/API/User/Activation.hs

```diff
@@ -158,6 +175,14 @@ data ActivationResponse = ActivationResponse
   }
   deriving stock (Eq, Show, Generic)
   deriving (Arbitrary) via (GenericUniform ActivationResponse)
+  deriving (ToJSON, FromJSON, S.ToSchema) via Schema ActivationResponse
+
+instance ToSchema ActivationResponse where
+  schema =
+    Schema.objectWithDocModifier "ActivationResponse" (description ?~ "Response body of a successful activation request") $
+      ActivationResponse
+        <$> activatedIdentity Schema..= userIdentityObjectSchema
+        <*> activatedFirst Schema..= (fromMaybe False <$> Schema.optFieldWithDocModifier "first" (description ?~ "Whether this is the first successful activation (i.e. account activation).") Schema.schema)
```

Comment:
```suggestion
        <*> activatedFirst Schema..= (fromMaybe False <$> Schema.optFieldWithDocModifier "first" (description ?~ "Whether this is the first successful activation of this account.") Schema.schema)
```

(Not sure I understand, though.  How often can one account be activated successfully?)

Reply:
This is from the original swagger documentation. IMO it should not be changed in this PR. Also it is not obvious to me either how this should be phrased correctly.